modules/exploits/linux/http/centreon_useralias_exec.rb

```diff
@@ -0,0 +1,104 @@
+##
+## This module requires Metasploit: http://metasploit.com/download
+## Current source: https://github.com/rapid7/metasploit-framework
+###
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'        => 'Centreon v2.5.3 Unauthenticated Command Execution',
```

Comment:
And no version either. Leave that to the description or a version field (if we ever get around to it). In general, ```Name``` should mimic the module name (or vice versa).